UM/Math/Color.py

```diff
@@ -67,3 +67,25 @@ def fromARGB(value):
             (value & 0x000000ff) >> 0,
             (value & 0xff000000) >> 24
         )
+
+    ##  Returns a new Color constructed from a 7-character string "#RRGGBB" format.
+    #
+    #   \param value A 7-character string representing a color in "#RRGGBB" format.
+    #   \return A Color constructed from the components of value.
+    @staticmethod
+    def fromRGBString(value):
```

Comment:
Sure, I could do that. I could also make it robust against 3 or 4 digit colors (eg #fff instead of #ffffff for white). I just did not want to add unnecessary complexity.